bigtable_input.py

```diff
@@ -169,24 +173,98 @@ def create(self):
         and their properties.
         """
         if self.bt_table.exists():
-            print('Table already exists')
+            utils.dbg('Table already exists')
             return
 
         max_versions_rule = column_family.MaxVersionsGCRule(1)
         self.bt_table.create(column_families={
             METADATA: max_versions_rule,
             TFEXAMPLE: max_versions_rule})
 
+    @property
     def latest_game_number(self):
         """Return the number of the next game to be written."""
-        game_counter = b'game_counter'
         table_state = self.bt_table.read_row(
             TABLE_STATE,
             filter_=row_filters.ColumnRangeFilter(
-                METADATA, game_counter, game_counter))
+                METADATA, GAME_COUNTER, GAME_COUNTER))
         if table_state is None:
             return 0
-        return cbt_intvalue(table_state.cell_value(METADATA, game_counter))
+        return cbt_intvalue(table_state.cell_value(METADATA, GAME_COUNTER))
+
+    @latest_game_number.setter
+    def latest_game_number(self, latest):
+        table_state = self.bt_table.row(TABLE_STATE)
+        table_state.set_cell(METADATA, GAME_COUNTER, int(latest))
+        table_state.commit()
+
+    def games_by_time(self, start_game, end_game):
+        """Given a range of games, return the games sorted by timestamp.
+
+        Returns [(timestamp, game), ...]
```

Comment:
where `game` is the row id?  what's the type of game?

Reply:
The game number, which is the basis of the row ID. Will clarify. The type of game (ordinary or calibration) depends on the `GameQueue` being used; the method is an instance method.